From ac5d3d644eff3cf11200e6a852a78e266e0370fa Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Thu, 12 Jun 2014 12:55:27 -0700
Subject: [PATCH 09/10] extcon: extcon-dc-pwrsrc: fix wrong value for charger
 detection

The DCP (dedicated charging port) is incorrectly defined as '2',
while the Dollar Cove PMIC output '3' for high current charger.

Change-Id: I6e320c2c0a03cd7e953941c463a2fd8fd5e00e36
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 drivers/extcon/extcon-dc-pwrsrc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/extcon/extcon-dc-pwrsrc.c b/drivers/extcon/extcon-dc-pwrsrc.c
index 66972f9..c8046bc2 100644
--- a/drivers/extcon/extcon-dc-pwrsrc.c
+++ b/drivers/extcon/extcon-dc-pwrsrc.c
@@ -74,7 +74,7 @@
 #define DET_STAT_POS			5
 #define DET_STAT_SDP			0x1
 #define DET_STAT_CDP			0x2
-#define DET_STAT_DCP			0x2
+#define DET_STAT_DCP			0x3
 
 #define DC_PS_BOOT_REASON_REG		0x2
 
-- 
2.0.0

